packages/jaeger-ui-components/src/TraceTimelineViewer/ListView/index.tsx

```diff
@@ -202,6 +202,10 @@ export default class ListView extends React.Component<TListViewProps> {
       }
       window.addEventListener('scroll', this._onScroll);
       this._windowScrollListenerAdded = true;
+    } else {
+      // The wrapper element should be the one that handles the scrolling. Once we are not using scroll-canvas we can remove this.
+      this._wrapperElm = document.getElementsByClassName('scroll-canvas')[0];
```

Comment:
Can we instead push this as a prop like `scrollElement?: DOMElement` or something like that from inside the explore/TraceView? Just to decouple this from the explore class names.

Reply:
Absolutely. 👌